config/settings.yml

```diff
@@ -1167,6 +1167,11 @@ lighthouse:
         client_id: ~
         rsa_key: ~
       use_mocks: false
+  benefits_documents:
+    host: https://sandbox-api.va.gov
```

Comment:
I believe we would want this to be the LH staging URL so we don't need to make devops changes in order to test in staging. Similar to the `form526` config

Reply:
Actually, per this conversation, https://dsva.slack.com/archives/C02CQP3RFFX/p1687972493754979, we should use sandbox. In this case the benefits docs API is actually wired up to the upstream services and LH's preference is that we use the sandbox uri

Reply:
Updated